### PR TITLE
Update use-group-policy.md

### PR DESCRIPTION
**Pull request by dariomws306**
https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/1378

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.
--

File: OneDrive/use-group-policy.md [hunk 1]
```diff
@@ -405,9 +405,9 @@ Please let us know if you have feedback on this feature or encounter any issues.
...
   
-If you enable this setting, new users who set up the sync client will download online-only files by default. If you disable this setting, Windows 10 users will have the same sync behavior as users of previous versions of Windows, and won't be able to turn on Files On-Demand. If you do not configure this setting, users can turn Files On-Demand on or off.
+If you enable this setting, new users who set up the sync client won't download online-only files by default. The files don’t download to the device until the are opened. If you disable this setting, Windows 10 users will have the same sync behavior as users of previous versions of Windows, and won't be able to turn on Files On-Demand. If you do not configure this setting, users can turn Files On-Demand on or off.
```
Comment (JohanFreelancer9) [inline OneDrive/use-group-policy.md]:
```suggestion
If you enable this setting, new users who set up the sync client won't download online-only files by default. The files don’t download to the device until they are opened. If you disable this setting, Windows 10 users will have the same sync behavior as users of previous versions of Windows, and won't be able to turn on Files On-Demand. If you do not configure this setting, users can turn Files On-Demand on or off.
```

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by kaarins** [comment]
Thanks for pointing out how the wording for this policy is unclear. I just merged a big change to the file, and I think it addresses the issue.

**Pull request closed without merging**